impl/maven-impl/src/main/java/org/apache/maven/impl/DefaultPluginXmlFactory.java

```diff
@@ -92,7 +93,7 @@ public void write(XmlWriterRequest<PluginDescriptor> request) throws XmlWriterEx
             } else if (outputStream != null) {
                 new PluginDescriptorStaxWriter().write(outputStream, content);
             } else {
-                try (OutputStream os = Files.newOutputStream(path)) {
+                try (OutputStream ignored = Files.newOutputStream(path)) {
```

Comment:
This doesn't look ignored. It looks like it's used?

Reply:
no its not:

<img width="1158" alt="image" src="https://github.com/user-attachments/assets/bd256a25-1ccc-4faf-aa09-fa2fdfc251cb" />
one time usage:

<img width="1055" alt="image" src="https://github.com/user-attachments/assets/0f1b9dac-2d67-496b-861a-4139147d94e8" />

`Value 'outputStream' is always 'null' `

<img width="1198" alt="image" src="https://github.com/user-attachments/assets/485d2b7c-3a42-4f26-b421-3bfbb5e239b7" />

this was the idea maybe:

<img width="1003" alt="image" src="https://github.com/user-attachments/assets/25e5704f-b7df-4c32-89fb-ad75c9b4e171" />